x-pack/legacy/plugins/lens/index.ts

```diff
@@ -64,6 +66,12 @@ export const lens: LegacyPluginInitializer = kibana => {
         savedObjects: server.savedObjects,
         config: server.config(),
         server,
+        taskManager: getTaskManagerSetup(server)!,
```

Comment:
two meta-issues here, perhaps:

1. the lens plugin doesn't list TM as a dep, but seems like it should
2. ideally, a plugin that uses telemetry shouldn't have to also directly use TM; feels like telemetry should handle this under the covers somehow

Reply:
Fair points, so:
1. This doesn't matter as much as you think, as all NP plugins are started before legacy plugins are initialised and you can't make a legacy plugin dependant on a NP one. That said, I'll add it so that when Lens get migrated to NP, this won't be forgotten. 👍 

2. I don't want to change how telemetry does things as part of this PR, as this is big enough with just the migration to NP. In anycase, I'm not sure I follow your thinking here, perhaps you should open an issue for it describing the problem you feel this creates?